### PR TITLE
Code of Conduct 

### LINKED ISSUE #2319: Adding Contributor Covenant Code of Conduct
### Is your feature request related to a problem? Please describe.

Suggestion.

### Describe the solution you'd like
Hey folks: can the project has https://github.com/internetarchive/iaux/blob/master/CODE_OF_CONDUCT.md added in the most contributed repository of internetarchive GitHub org?

### Proposal & Constraints
https://github.com/internetarchive/iaux/blob/master/CODE_OF_CONDUCT.md

### Additional context
None.

### PR DESCRIPTION
**Pull request by tabshaikh**
### Description

Closes #2319

Code of conduct added and `Instances of abusive, harassing, or otherwise unacceptable behavior it would be reported to openlibrary@archive.org` as discussed in #team-abc meeting and then it would be forwarded to the respective hr.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at openlibrary@archive.org. All
```
Comment (tfmorris) [inline CODE_OF_CONDUCT.md]: Should this be a separate dedicated address?

**Comment by cdrini** [comment]
@tabshaikh please avoid assigning others to PRs; or justify why you are doing so.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (cdrini) [reply CODE_OF_CONDUCT.md]: Is that the same email we use for the "contact us" page? If so, it probably should be a separate email.

**Comment by cdrini** [comment]
@tfmorris abc is the team of employed developers at the internet archive that work on open library. It include Mek, Charles, myself, and Tabish (since he was employed there during Google Summer of Code).

**Comment by cdrini** [comment]
@tabshaikh Also note the issue create for this: #2319 ; if you haven't seen it yet, please make sure to read the comments there.

**Comment by mekarpeles** [comment]
After community discussion, we absolutely want a(n enforceable) code of conduct and are opting to punt until after October madness so we can get input from our HR and openlibrary staff on how to proceed.

**Comment by tfmorris** [comment]
Happy to defer this for a bit, but I don't think the OpenLibrary community guidelines need to be done in lock step with what Internet Archive's HR department mandates for their employees. The community includes many open data and open source advocates who aren't employed by or affiliated in any way with Internet Archive.

**Pull request closed without merging**